model/resource/service_experimental.yaml

```diff
@@ -22,16 +22,116 @@ groups:
+          confidential by this algorithm, being the user's choice to expose it or not via another resource attribute.
+
+          When a UUID v5 is generated, the input MUST be prefixed with
+          `${telemetry.sdk.name}.${telemetry.sdk.language}.${service.namespace}.${service.name}`, followed by the
+          the workload identifier, which should tentatively be stable. The workload identifier is the final piece of
+          information making the source of telemetry to be uniquely idenfiable. For instance, it might be the `container.id`
+          or `container.name` in containerized or Kubernetes environments. This means that the same service yields the
+          same UUID if the same identifier (`host.id`, `/etc/machine-id`, and so on) remains the same. It would still
+          yield different results for different services on the same host or namespace. When no namespaces or equivalent fields
+          are available, the prefix MUST then be `${telemetry.sdk.name}.${telemetry.sdk.language}.${service.name}`.
+
+          Users are encouraged to explicitly set the `service.instance.id` themselves, given they are in the best position
+          to determine what's a service instance. For instance, in some cases, all containers in a given pod might be part
+          of the same service, while in other cases, each container would independently generate their own telemetry. When
+          it's not possible or desirable to explicitly generate a `service.instance.id`, users are encouraged to set
+          enough information via their workload automation, such as Helm. In that case, SDKs would have enough data to
+          generate a `service.instance.id`. Future versions of the OpenTelemetry Operator might also have the ability to
+          generate the `service.instance.id` and set container-specific labels, given it is in a position to know all that
+          metadata. When SDKs don't have sufficient data to generate a stable UUID, a random one is generated.
+
+          In more concrete terms: users running their services on platforms such as Kubernetes are encouraged to explicitly
+          set the `service.instance.id` using their existing automation, or set a value that can be used for a
+          consistent value, such as `container.id`. Similarly, users of application servers such as `unicorn`
+          are encouraged to set the `service.instance.id` on a per-worker basis. SDKs MUST use the following algorithm when
+          generating `service.instance.id`:
+
+          - If the user has provided a `service.instance.id`, via environment
+            variable, configuration or custom resource detection, this MUST take priority over generated IDs.
+          - When any of the below combinations of resource attribute are provided, they MUST be used as the input
+            for generating a UUID v5 following the prefix mentioned above. The values within each combination MUST be separated with dots:
```

Comment:
Is there a link describing how it works? I would like to point people to there for reference.

Reply:
https://github.com/open-telemetry/opentelemetry-erlang/blob/main/apps/opentelemetry/src/otel_resource_detector.erl#L249

Reply:
TL;DR; it gets the contents of [`erlang:node()`](https://www.erlang.org/doc/reference_manual/distributed.html#nodes) as long as that value is not `noname@nohost` or `localhost`.

Reply:
Am I right to assume that it will use the host as it is, without hashing it in some way?

https://github.com/open-telemetry/opentelemetry-erlang/blob/82c090f2c9b4d91b84d59bfd1385d1b188a01ec6/apps/opentelemetry/src/otel_resource_detector.erl#L263